### ISSUE TITLE
fragment fade-in-then-out not working

### ISSUE THREAD

**Issue by OmarAshkar** [rstudio/revealjs#73]
There is one class in revealjs that I am intereseted in called "fragment fade-in-then-out" as shown here in the documentation

https://github.com/hakimel/reveal.js/blob/3643112b35255b2fddf21041343b31cc64222fb1/README.md#fragments

in my rmarkdown file

<div class="fragment fade-in-then-out" style="position: absolute">
text 66
</div>

<div class="fragment fade-in-then-out" style="position: absolute">
text 1
</div>

resulted in fading in, but there is no out on the second press.

**Comment by apreshill** [comment]
With upgraded reveal.js via Pandoc: I can see this effect with:

```{=html}
<p class="fragment">Fade in</p>
<p class="fragment fade-out">Fade out</p>
<p class="fragment highlight-red">Highlight red</p>
<p class="fragment fade-in-then-out">Fade in, then out</p>
<p class="fragment fade-up">Slide up while fading in</p>
```

Code from: https://revealjs.com/fragments/

The same code in line 4 does fade in but not out with `revealjs::revealjs_presentation` installed from Github.

**Comment by cderv** [comment]
Regarding the non working fragment, this is a matter of reveal.js version. We have this: 
https://github.com/rstudio/revealjs/blob/ee7e9b5dfe18639648513c9bc4a2cba59461f95b/inst/reveal.js-3.3.0.1/README.md#L649-L666
* `fade-in-then-out` was a renaming of `current-visible` in 3.7 (https://github.com/hakimel/reveal.js/commit/4bea8e17e8a6dd1580286aa3b0b927e0aebae75b)
* `fade-up` is available since 3.3 (https://github.com/hakimel/reveal.js/commit/ee877845d8c1c03b55d218f0f28b352c52ccba6e) so it should work

I tried this and it works better, excep `fade-up` 🤷 

Let's try

````html
<p class="fragment">Fade in</p>
<p class="fragment fade-out">Fade out</p>
<p class="fragment highlight-red">Highlight red</p>
<p class="fragment current-visible">Fade in, then out</p>
<p class="fragment fade-up">Slide up while fading in</p>
````

Answer to this issue is to use `.current-visible` class and then it will be working. 

I'll open another one about the fragment syntax improvement
